changelog.py

```diff
@@ -150,31 +150,39 @@ def main():
     parser.add_argument("--token", type=str, help="github token")
     parser.add_argument("--login", type=str, help="github login")
     parser.add_argument("--password", type=str, help="github password")
+    parser.add_argument("--list", type=int, nargs=2, metavar=('first', 'last'),
```

Comment:
This is more of a theoretical question, but I wonder is it a "good pattern" of "flags" usage? I always thought about them as switches to alter a command behavior. They are subcommands by themselves here. At least some of them, e.g. you can use `--token` with `--list` but not `-c` with `--list` which is not obvious without reading the code.